packages/components/src/components/domainproperties/issues/IssuesListDefDesignerPanels.tsx

```diff
@@ -0,0 +1,192 @@
+                        setSubmitting(false, () => {
+                            const { model } = this.state;
+                            this.props.onComplete(model);
+                    })
+                });
+            })
+            .catch(response => {
+                const exception = resolveErrorMessage(response);
+
+                this.setState(
+                    produce((draft: Draft<State>) => {
+                        const updatedModel = draft.model;
+                        if (exception) {
+                            updatedModel.exception = exception;
+                        } else {
+                            updatedModel.exception = undefined;
+                            updatedModel.domain = response;
+                        }
+                    }),
+                    () => {
+                        setSubmitting(false, () => {
+                            this.props.onComplete(model);
+                        })
+                    }
+                )
+            });
+    };
+
+    render() {
+        const {
+            onCancel,
+            useTheme,
+            containerTop,
+            successBsStyle,
+            visitedPanels,
+            currentPanelIndex,
+            firstState,
+            validatePanel,
+            submitting,
+            onTogglePanel,
+            saveBtnText,
+        } = this.props;
+        const { model } = this.state;
+
+        return (
+            <BaseDomainDesigner
+                name={model.name}
+                exception={model.exception}
+                domains={List.of(model.domain)}
+                hasValidProperties={model.hasValidProperties()}
+                visitedPanels={visitedPanels}
+                submitting={submitting}
+                onCancel={onCancel}
+                onFinish={this.onFinish}
+                saveBtnText={saveBtnText}
+                successBsStyle={successBsStyle}
+            >
+                <IssuesListDefPropertiesPanel
+                    model={model}
+                    onChange={this.onPropertiesChange}
+                    controlledCollapse={true}
+                    initCollapsed={currentPanelIndex !== 0}
+                    panelStatus={
+                        model.isNew()
+                            ? getDomainPanelStatus(0, currentPanelIndex, visitedPanels, firstState)
+                            : 'COMPLETE'
+                    }
+                    validate={validatePanel === 0}
+                    onToggle={(collapsed, callback) => {
+                        onTogglePanel(0, collapsed, callback);
+                    }}
+                    useTheme={useTheme}
+                    successBsStyle={successBsStyle}
+                />
+                <DomainForm
+                    key={model.domain.domainId || 0}
+                    domainIndex={0}
+                    domain={model.domain}
+                    headerTitle="Fields"
+                    helpNoun="list"
```

Comment:
not sure what value we want this to be for "issue lists" but looks like this value of "list" was likely copy-pasted from `ListDesignerPanels.tsx

Reply:
yup, copy-paste from list.
now modified.